src/Analysis/Ast/Impl/Analyzer/Handlers/ImportHandler.cs

```diff
@@ -56,19 +56,22 @@ internal sealed partial class ImportHandler : StatementHandler {
             // import_module('fob.oar.baz')
             var importNames = ImmutableArray<string>.Empty;
             var variableModule = default(PythonVariableModule);
+            var importedNamesCount = 0;
             foreach (var nameExpression in moduleImportExpression.Names) {
                 importNames = importNames.Add(nameExpression.Name);
                 var imports = ModuleResolution.CurrentPathResolver.GetImportsFromAbsoluteName(Module.FilePath, importNames, forceAbsolute);
                 if (!HandleImportSearchResult(imports, variableModule, asNameExpression, moduleImportExpression, out variableModule)) {
-                    return;
+                    break;
```

Comment:
This handles a case when `import package.subpackage.module` fails on `subpackage` part, but `package` has `__init__.py` and successfully imported.